kube/int/create.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -ex
-DAX_NAMESPACE=${DAX_NAMESPACE:-'dax-int'}
+DAX_NAMESPACE=${DAX_NAMESPACE:-'lsst-lsp-int-dax'}
 
 # kubectl create -f dax-metaserv-datasets-volume.yaml
```

Comment:
Please get rid of commented lines here and PVs.  Metaserv doesn't need them, and will never need them.

Reply:
Done.